### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Add a resource via Azure Active Directory_Attach with Azure AD_Non-text Contrast: When the "Resource type" dropdown receives the keyboard focus, the color contrast ratio of the border with respect to background is less than 3:1.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/AzureStorageExplorer#3462]
**GitHubTags:** #C&AI_Aug2020_BMAzure Tools Storage Explorer; #A11yWCAG2.1; #A11YTCS; #DesktopApp; #Azure Tools Storage Explorer; #WCAG1.4.11; #Win32; #A11ySev2; #Benchmark; #ColorContrast;#AI4D;#FTP;#STP

### Environment Details:
Application Name: Azure Tool Storage Explorer
Application version: 1.14.2
Windows Version - Windows10

### Repro Steps:

1. Launch Storage Explorer
2. Navigate to "Open Connect Dialog" pane present on left side of screen and hit enter.
3. Select "Add a resource via Azure Active Directory(Azure AD)" radio button and hit enter on Next button.
4. "Attach with azure AD" screen gets opened and navigate to Next button and hit enter.
5. Navigate till "Resource type" dropdown using tab key.
6. Open AI4D and check the color contrast ratio of the border with respect to the background when "Resource type" dropdown receives the keyboard focus.

### Actual:
When the "Resource type" dropdown receives the keyboard focus, the color contrast ratio of the border with respect to background is less than 3:1.
**This issue is observed for all the dropdown present in the application.**

### Expected:
When the "Resource type" dropdown receives the keyboard focus, the color contrast ratio of the border with respect to background should be equal or greater than 3:1

### User Impact:
Low vision users may not able to see the control has keyboard focus.

### Attachment: 
https://microsoftapc-my.sharepoint.com/:i:/g/personal/v-shwarh_microsoft_com/EdbDFThH-qdFuUlHDOpe-xQBRfTrWCwTtJNajPvSJVc-ag?e=USC1Dj

**Comment by MRayermannMSFT** [comment]
![Uploading image.png…]()

**Comment by 12-shweta** [comment]
Verified the bug in the version 1.16.0-dev
The issue is Fixed and does not reproduce.
Hence closing this issue.
Please find the below link for Closing screenshot
https://microsoftapc-my.sharepoint.com/:i:/g/personal/v-shwarh_microsoft_com/EU7dOyBvHO5EihbpvRyCJ-kByv56t-se7ir5Pq5h_KIfag?e=MBepZO

**Comment by fsteffi** [comment]
**GitHub Tags:**#A11yMAS;

**Comment by 12-shweta** [comment]
Github Tags:#Closed;
